### ISSUE TITLE
tridesclous GUI crashes when opening Blackrock data file

### ISSUE THREAD

**Issue by vncntprvst** [tridesclous/tridesclous#28]
After opening the GUI, then File>Initialize dataset>source_type:Blackrock>Add file
tried opening either .NEV or .NS6 files. Both induced the GUI to crash. 

First case, with .NEV:
`on_addfiles
<class 'tridesclous.datasource.BlackrockDataSource'>
multi-file
source_params {'filenames': ['C:/Data/raw/Blackrock/031S_1103/031S_1103_WR_MN.ne
v']}
['C:/Data/raw/Blackrock/031S_1103/031S_1103_WR_MN.nev']
Traceback (most recent call last):
  File "C:\Anaconda\envs\tridesclous\lib\site-packages\tridesclous-0.2.0.dev0-py
3.6.egg\tridesclous\gui\initializedatasetwindow.py", line 163, in validate_step
  File "C:\Anaconda\envs\tridesclous\lib\site-packages\tridesclous-0.2.0.dev0-py
3.6.egg\tridesclous\gui\initializedatasetwindow.py", line 100, in display_step
  File "C:\Anaconda\envs\tridesclous\lib\site-packages\tridesclous-0.2.0.dev0-py
3.6.egg\tridesclous\datasource.py", line 179, in __init__
  File "C:\Anaconda\envs\tridesclous\lib\site-packages\tridesclous-0.2.0.dev0-py
3.6.egg\tridesclous\datasource.py", line 145, in channel_ids
KeyError: 5`

Second case, with .NS6:
`on_addfiles
<class 'tridesclous.datasource.BlackrockDataSource'>
multi-file
source_params {'filenames': ['C:/Data/raw/Blackrock/031S_1103/031S_1103_WR_MN.ns
6']}
['C:/Data/raw/Blackrock/031S_1103/031S_1103_WR_MN.ns6']
Traceback (most recent call last):
  File "C:\Anaconda\envs\tridesclous\lib\site-packages\tridesclous-0.2.0.dev0-py
3.6.egg\tridesclous\gui\initializedatasetwindow.py", line 163, in validate_step
  File "C:\Anaconda\envs\tridesclous\lib\site-packages\tridesclous-0.2.0.dev0-py
3.6.egg\tridesclous\gui\initializedatasetwindow.py", line 100, in display_step
  File "C:\Anaconda\envs\tridesclous\lib\site-packages\tridesclous-0.2.0.dev0-py
3.6.egg\tridesclous\datasource.py", line 177, in __init__
  File "C:\Anaconda\envs\tridesclous\lib\site-packages\neo\io\blackrockio.py", l
ine 271, in __init__
    'No Blackrock files present at {}'.format(filename))
OSError: No Blackrock files present at C:/Data/raw/Blackrock/031S_1103/031S_1103
_WR_MN.ns6`

**Comment by samuelgarcia** [comment]
Wich version of ne do you have ?
To fix this you can install neo from master branch (0.6 dev).

I will have a look for neo 0.5

**Comment by samuelgarcia** [comment]
I have done some test with neo 0.5.2 with a ns5 sile and it is working.
Maybe this is due to ns6. What is the difference with ns5 ?
Could you provide me a small datasets with ns6, so I could test it ?

**Comment by vncntprvst** [comment]
.ns5 is filtered (if a filter is selected) data, sampled at 30kHz
.n6 is raw signals, also sampled at 30kHz
You can access an example recording set here:
https://duke.box.com/s/junq00niupy1johzftqdkzfythy5jn56
Thanks!

**Comment by vncntprvst** [comment]
I upgraded neo to 0.6, updated and re-installed tdc, but still get the same crash when opening a Blackrock file. 
`on_addfiles
<class 'tridesclous.datasource.BlackrockDataSource'>
multi-file
source_params {'filenames': ['C:/Data/raw/Blackrock/039v_1011/039v_1011_WheelRun
ning_LaserStim.ns6']}
['C:/Data/raw/Blackrock/039v_1011/039v_1011_WheelRunning_LaserStim.ns6']
Traceback (most recent call last):
  File "C:\Anaconda\envs\tridesclous\lib\site-packages\tridesclous-0.2.0.dev0-py
3.6.egg\tridesclous\gui\initializedatasetwindow.py", line 163, in validate_step
  File "C:\Anaconda\envs\tridesclous\lib\site-packages\tridesclous-0.2.0.dev0-py
3.6.egg\tridesclous\gui\initializedatasetwindow.py", line 100, in display_step
  File "C:\Anaconda\envs\tridesclous\lib\site-packages\tridesclous-0.2.0.dev0-py
3.6.egg\tridesclous\datasource.py", line 177, in __init__
  File "C:\Anaconda\envs\tridesclous\lib\site-packages\neo\io\blackrockio.py", l
ine 271, in __init__
    'No Blackrock files present at {}'.format(filename))
OSError: No Blackrock files present at C:/Data/raw/Blackrock/039v_1011/039v_1011
_WheelRunning_LaserStim.ns6`

**Comment by samuelgarcia** [comment]
For neo 0.5.2 it was a mistake I hard coded to take ns5 in trdesclous.
This is fixed with PR #32.

For neo 0.6 (unreleased) master branch. The file you send are with tridesclous.

I suspect with the error you send that you still have neo 0.5.2 because it refer neo.io and now it use neo.rawio (0.6).

You should do
```
pip uninstall neo
pip install https://github.com/NeuralEnsemble/python-neo/archive/master.zip
```

**Comment by vncntprvst** [comment]
Nice. That works.
